### PR TITLE
OCP nodes ssh timeout option

### LINKED ISSUE #17: Make cluster node ssh timeout configurable
Need to provide an option to the playbook for user specified timeout for OCP nodes ssh connection.
Default timeout need to be increased from 30m to 45m.

### PR DESCRIPTION
**Pull request by yussufsh**
Fixes #17

Signed-off-by: Yussuf Shaikh <yussuf@us.ibm.com>

### CODE REVIEW

**Review by Prajyot-Parab — DISMISSED** [review]
lgtm!

File: examples/install_vars.yaml [hunk 1]
```diff
@@ -10,10 +10,12 @@ storage_type: nfs
 log_level: info
 release_image_override: ""
 
+node_connection_timeout: 2700
+
 rhcos_kernel_options: []
+sysctl_tuned_options: false
```
Comment (bpradipt) [inline examples/install_vars.yaml]: hmm.. why this entry for timeout option ?

Comment (yussufsh) [reply examples/install_vars.yaml]: Just added the missing variable in example.

Comment (bpradipt) [reply examples/install_vars.yaml]: yeah.. better to split this in a new PR and you can put the message as adding missing variable in examples

Comment (yussufsh) [reply examples/install_vars.yaml]: ok

**Review by Prajyot-Parab — APPROVED** [review]

**Review by bpradipt — APPROVED** [review]
lgtm

**Pull request merged**